nanomq/sub_handler.c

```diff
@@ -302,17 +302,6 @@ sub_ctx_handle(nano_work *work)
 			tq = tq->next;
 		}
 		if (!topic_exist && topic_str) {
-			int            t = 0;
-			uint8_t ver = conn_param_get_protover(work->cparam);
```

Comment:
Why this is deleted,  it caused compile error.

Reply:
I am so sorry. I deleted the lines by mistake. Maybe you can recover it and move the lines to the start of this function.